### ISSUE TITLE
NMS is not able to work with Postgres 9.4+

### ISSUE THREAD

**Issue by 119Vik** [magma/magma#2896]
### Environment

- Magma 1.2
- NMS
- @fbcnms/sequelize-models/index.js

### Describe the Issue

In case NMS deployed with Postgres DB each attempt to login leads to failure.
In scope of auth/express.js following SQL request is generated :
`**SELECT "id", "name", "tabs", "csvCharset", "customDomains", "networkIDs", "ssoSelectedType", "ssoCert", "ssoEntrypoint", "ssoIssuer", "ssoOidcClientID", "ssoOidcClientSecret", "ssoOidcConfigurationURL", "createdAt", "updatedAt" FROM "Organizations" AS "Organization" WHERE (SELECT 1 FROM json_each("customDomains")WHERE json_each.value = 'master.nms.yourdomain.com') IS NOT NULL LIMIT 1;**`
Following error is raised:
`[auth/express.js] error: Error getting organization operator does not exist: json = unknown`

Bug is in the function which responsible for jsonArrayContains check
```
export function jsonArrayContains(column: string, value: string) {
  if (
    sequelize.getDialect() === 'mysql' ||
    sequelize.getDialect() === 'mariadb'
  ) {
    return Sequelize.fn('JSON_CONTAINS', Sequelize.col(column), `"${value}"`);
  } else {
    // sqlite
    const escapedColumn = sequelize
      .getQueryInterface()
      .quoteIdentifier(column, true);
    const innerQuery = Sequelize.literal(
      **`(SELECT 1 FROM json_each(${escapedColumn})` +
        `WHERE json_each.value = ${sequelize.escape(value)})`,**
    );
    return Sequelize.where(innerQuery, 'IS', Sequelize.literal('NOT NULL'));
  }
}
```

**Expected behavior**
NMS works with Postgres as good as with MySQL

For the Postgres following query will work:
**SELECT “id”, “name”, “tabs”, “csvCharset”, “customDomains”, “networkIDs”, “ssoSelectedType”, “ssoCert”, “ssoEntrypoint”, “ssoIssuer”, “ssoOidcClientID”, “ssoOidcClientSecret”, “ssoOidcConfigurationURL”, “createdAt”, “updatedAt” FROM “Organizations” AS “Organization” WHERE “customDomains”::jsonb ? ‘master.nms.yourdomain.com’;**

**Comment by karthiksubraveti** [comment]
@mattymo I'm in favor of closing this issue since we have other issues https://github.com/magma/magma/issues?q=is%3Aopen+is%3Aissue+author%3Aandreilee+postgres tracking this. Thoughts ?

**Comment by mattymo** [comment]
As long as it starts working, I don't care if you close this. I hope you and @andreilee have success getting it done.

**Comment by andreilee** [comment]
@mattymo @karthiksubraveti Asides from the duplicate issue that exists, this has been nearly fixed. The upstream dependency `fbcnms-sequelize-models` has had the function `jsonArrayContains` fixed to support Postgres, and that's available in `^0.1.1`. The linked PR puts that version as a requirement for Magma NMS.
